src/cloudai/_core/report_generation_strategy.py

```
@@ -30,7 +30,7 @@ def __init__(self, system: System, tr: TestRun) -> None:
         self.system = system
         self.test_run = tr
 
-    def get_metric(self, metric: str) -> float:
+    def get_metric(self, metric: str) -> MetricValue:
```

Ruff (0.15.9)

[warning] Unused method argument: `metric`

(ARG002)